docs/proposals/20240807-in-place-updates.md

```diff
@@ -697,7 +712,7 @@ spec:
 +       name: md-1-2
 ```
 
-The `vsphere-vm-memory-update` extension informs that can cover all requested changes:
+The `vsphere-vm-memory-update` extension informs that can cover required requested changes:
```

Comment:
same as before, this is not done by the extension
let's talk about this offline and make all related changes at once. i think we need to change everything that touches the "deciding the update strategy" and "checking desired changes coverage"